### PR TITLE
Ring of Khajiiti traitor item

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Adds the Ring of Khajiiti (because of course we need more TES items), which is essentially a better version of the thief gloves, with (sightly) faster pickpocketing and heat protection as well as insulation.

## Why It's Good For The Game

Khajiiti has wares... if you have telecrystal.

## Changelog
:cl:
add: Added the Ring of Khajiiti as a traitor item.
code: The stealth suit is now in the "clothing" section of the uplink.
/:cl:

### CODE REVIEW

**Comment by BobJoga** [comment]
it compiles

File: modular_skyrat/code/modules/uplink/uplink_clothing.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/uplink_item/suits/stealthsuit
+	name = "Tactical Stealth Suit"
+	desc = "A tight-fitting stealth suit, able to make you near completely invisible."
+	item = /obj/item/clothing/under/syndicate/stealthsuit
+	cost = 5
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/uplink/uplink_clothing.dm]: Is this supposed to be a part of this?

Comment (jjpark-kb) [reply modular_skyrat/code/modules/uplink/uplink_clothing.dm]: Ah, I see you have it a part of the code section

Comment (BobJoga) [reply modular_skyrat/code/modules/uplink/uplink_clothing.dm]: I moved it to clothing because it makes more sense.

**Comment by Azarak** [comment]
camoflaged thieving gloves that are also combat gloves? Please no, also slow down with the references to TEG or stuff, we're a roleplaying server in a sci-fi universe

**Comment by BobJoga** [comment]
> camoflaged thieving gloves that are also combat gloves? Please no, also slow down with the references to TEG or stuff, we're a roleplaying server in a sci-fi universe

but i like them
also this costs more than the standard thieving gloves
and standard thieving gloves are easily noticeable by powergamers because of the white fingertips on the sprites

**Pull request closed without merging**